.github/dockerfiles/Dockerfile.debian-base

```diff
@@ -11,7 +11,7 @@ ENV HOST_TRIP=$HOST_TRIP
 ENV LANG=C.UTF-8
 ENV LC_ALL=C.UTF-8
 
-ENV INSTALL_LIBS="zlib1g-dev libncurses5-dev libssl-dev unixodbc-dev libgmp3-dev libwxgtk3.0-gtk3-dev libwxgtk-webview3.0-gtk3-dev libsctp-dev lksctp-tools"
+ENV INSTALL_LIBS="zlib1g-dev libncurses5-dev libssl-dev unixodbc-dev libgmp3-dev libwxgtk3.0-gtk3-dev libwxgtk-webview3.0-gtk3-dev libwebkit2gtk-4.0-dev libwebkit2gtk-4.0-37 gir1.2-webkit2-4.0 libjavascriptcoregtk-4.0-dev libjavascriptcoregtk-4.0-18 gir1.2-javascriptcoregtk-4.0 libsctp-dev lksctp-tools"
```

Comment:
would be nice if we have this change documented (maybe in the git commit message) @thalesmg ?
https://github.com/emqx/otp/pull/25/commits/f1fe6a78a89893a6bd47b3db13940ad925e2148a

Reply:
From what I can recall (and from the commit message you found), it was my attempt at fixing CI after it broke without touching the C stuff, and upstream was not updated yet, I think.  So I tried to search for which packages needed upgrading to make it work.  :thinking: